src/procedures/ModifyTieredStoData.ts

```diff
@@ -33,9 +36,35 @@ export const createTieredStoFactoryRefreshResolver = (
   return undefined;
 };
 
+/**
+ * Procedure to modify data associated to a Tiered STO.
+ * Transactions include times, funding type, oracles, tiers, limits, and addresses
+ */
 export class ModifyTieredStoData extends Procedure<ModifyTieredStoDataProcedureArgs> {
   public type = ProcedureType.ModifyTieredStoData;
 
+  /**
+   * - Modify the start and end date of the STO
+   *
+   * - Modify the fundraise types of the STO, whether they be ETH, POLY, or Stablecoin
+   *
+   * - Modify the oracles of the STO including the denominated currency, as well the POLY and ETH oracle addresses
```

Comment:
`- Modify the custom currency symbol and the oracles used to convert POLY and ETH to said currency`